test/service/ConfigLoader.test.ts

```diff
@@ -33,6 +34,24 @@ class ConfigLoaderMocked extends ConfigLoader {
 }
 
 describe('ConfigLoader', () => {
+    it('ConfigLoader loadPresetData no preset', async () => {
+        const configLoader = new ConfigLoaderMocked(logger);
+        try {
+            await configLoader.createPresetData({
+                preset: undefined,
+                assembly: undefined,
+                customPreset: undefined,
+                customPresetObject: undefined,
+                password: 'abc',
+            });
+            expect(false).to.be.eq(true);
```

Comment:
you might want to include a message for these

Reply:
Added. Note that `expect(()=>someFunction()).to.throw` does not work with async functions. I didn't want to add a new library just for this.